test/benchmarks/benchmark_chunkmemset.cc

```
@@ -0,0 +1,127 @@
+
+#define BUFSIZE        8192
+#define BENCH_MAX_DIST 64
+
+typedef uint8_t* (*chunkmemset_safe_func)(uint8_t *out, uint8_t *from, size_t len, size_t left);
+
+class chunkmemset: public benchmark::Fixture {
+private:
+    uint8_t *buf;
+
+public:
+    void SetUp(::benchmark::State& state) {
```

Cppcheck (2.20.0)

[style] The function 'SetUp' is never used.

(unusedFunction)